modules/api/src/main/java/org/eclipse/xpanse/api/config/ServiceTemplateEntityConverter.java

```diff
@@ -100,6 +102,8 @@ public static UserOrderableServiceVo convertToUserOrderableServiceVo(
             userOrderableServiceVo.setServiceAvailabilityConfig(
                     serviceTemplateEntity.getOcl().getDeployment().getServiceAvailabilityConfig());
             userOrderableServiceVo.setEula(serviceTemplateEntity.getOcl().getEula());
+            userOrderableServiceVo.setConfigurationParameters(serviceTemplateEntity.getOcl()
+                    .getServiceConfigurationManage().getConfigurationParameters());
```

Comment:
The filed `serviceConfigurationManage`  is not a NotNull field in `Ocl` object. Need to check null to prevent null pointer exception

Reply:
Fixed.